core/charm/repository/charmhub.go

```diff
@@ -389,26 +387,16 @@ func (c *CharmHubRepository) ListResources(charmURL *charm.URL, origin corecharm
 // a slice with the results. The results include the minimum set of metadata
 // that is required for deploying each charm.
 func (c *CharmHubRepository) GetEssentialMetadata(reqs ...corecharm.MetadataRequest) ([]corecharm.EssentialMetadata, error) {
+	// TODO (cderici): This grouping below might not be necessary anymore
+	// since we don't have any macaroons anymore
+
```

Comment:
unrelated, couldn't select the lines for a pr comment - there is a TODO to be removed: lines 424-428